apollo-router/src/services/http.rs

```diff
@@ -21,13 +21,13 @@ pub(crate) type ServiceResult = Result<HttpResponse, BoxError>;
 
 #[non_exhaustive]
 pub(crate) struct HttpRequest {
-    pub(crate) http_request: http::Request<Body>,
+    pub(crate) http_request: http::Request<RouterBody>,
```

Comment:
Breaking change, but ok because we haven't made this publicly useable.

Reply:
why would it be a breaking change if it is not public?

Reply:
It was a note to myself to make sure I reviewed usage of HttpRequest carefully. I should have removed it before I finished the review. Sorry for the noise.